src/main/resources/templates/db_scripts/schema.json

```diff
@@ -69,6 +75,48 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "contacts",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        },
+        {
+          "fieldName": "erpCode",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        },
+        {
+          "fieldName": "interfaces",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        },
+        {
+          "fieldName": "language",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": true
+        },
+        {
+          "fieldName": "name",
```

Comment:
Field "name" is already declared https://github.com/folio-org/mod-organizations-storage/pull/31/files#diff-162ca424ba9ad000edded275876b325bR68

Reply:
Thanks, good catch! I've removed the duplicate name, now all is sorted alphabetically.